x-pack/legacy/plugins/ml/public/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -882,7 +883,8 @@ export class TimeSeriesExplorer extends React.Component {
 
     timefilter.enableTimeRangeSelector();
     timefilter.enableAutoRefreshSelector();
-    timefilter.on('timeUpdate', this.refresh);
+
+    this.timeUpdateSubscription = timefilter.getTimeUpdate$().subscribe(this.refresh);
```

Comment:
This could be `subscriptions.add(timefilter.getTimeUpdate$().subscribe(this.refresh));`

Reply:
done